src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="page">
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
Check [BEM methodology ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Modifier classes should not be used on their own. Instead, they should be used in conjunction with the block or element class they are modifying (`block__element--modifier`).